spec.md

```diff
@@ -377,24 +377,32 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+A Platform will find it useful for the Broker to define exactly what is expected
+when making requests inside opaque input request objects and unspecified output
+responses, that mechanism is JSON Schema Object definitions. This allows Broker
+authors to programmatically define inputs and outputs for applicable requests.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| parameters | JSON schema object | The schema definition for the input parameters. Each input parameter is expressed as a property within a JSON object. |
+| parameters | JSON schema object | The schema definition for the input `parameters`. Each input parameter is expressed as a property within a JSON object. |
+| response | JSON schema object | The schema definition for the output response. |
```

Comment:
Where do you think we may eventually add conventions for output responses? Ideally we would be able to encourage consistency across brokers for the common things platforms need to do when creating a binding i.e. changing networking/firewall policies.

Reply:
>add conventions for output responses

I think that process should be the same process for agreement on generic actions. It is very much the same kind of community lead standardization. So... we just have to agree on how to get the generic actions out ! :D